source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -295,7 +295,7 @@ void RedisCluster::RedisDiscoverySession::onResponse(
   const uint32_t SlotRangeStart = 0;
   const uint32_t SlotRangeEnd = 1;
   const uint32_t SlotMaster = 2;
```

Comment:
`SlotPrimary`?

Reply:
I will do a follow up for redis. It's a larger change.